htdocs/web_portal/img/README.md

```diff
@@ -1,22 +1,25 @@
-|star.png|[Link](http://www.softicons.com/web-icons/web-2-icon-set-by-anders-bjarnle/star-icon)|Free Use of icon|Anders Bjarnle|
+|key.png|[Link](http://www.softicons.com/toolbar-icons/build-icons-by-design-kindle/key-icon)|Free for commercial use|	Design Kindle|
+|network.png|[Link](https://findicons.com/icon/169709/globe)|GNU Lesser General Public License|YellowIcon|
+|new_window.png|[Link](http://www.softicons.com/toolbar-icons/cyan-bitcons-icons-by-some-random-dude/new-window-icon)|CC Attribution-ShareAlike 3.0 United States|Some Random Dude (<-
+actually the author information)|
 |ngi.png|[Link](https://findicons.com/icon/234318/package_network)|GNU Lesser General Public License|Pavel InFeRnODeMoN|
-|virtualsite.png|[Link](http://www.softicons.com/android-icons/mobile-icon-set-by-webiconset.com/maps-icon)|Free Use of icon|	WebIconSet.com|
-|new_window.png|[Link](http://www.softicons.com/toolbar-icons/cyan-bitcons-icons-by-some-random-dude/new-window-icon)|CC Attribution-ShareAlike 3.0 United States|Some Random Dude (<- actually the author information)|
 |pencil.png|[Link](http://www.softicons.com/toolbar-icons/matte-basic-icons-by-momentum/edit-add-icon)|CC Attribution-Share Alike 3.0 United States|Momentum|
+|person.png|[Link](http://www.iconarchive.com/show/ecommerce-business-icons-by-designcontest/admin-icon.html)|CC Attribution 4.0|DesignContest|
 |pin.png|[Link](http://www.softicons.com/system-icons/windows-8-metro-invert-icons-by-dakirby309/pin-icon)|CC Attribution 3.0 Unported|	dAKirby309|
-|down_arrow.png|[Link](http://www.softicons.com/toolbar-icons/marmalade-icons-by-icojam/arrow-down-icon)|Freeware commercial license|	Icojam|
+|project.png|[Link](http://www.icons101.com/icon/id_51397/setid_1789/Free_3D_Printer_by_AhaSoft/Project)|CC Attribution|Aha-Soft (website)|
+|search.png|[Link](http://www.softicons.com/business-icons/business-icons-by-petalart/search-icon)|CC Attribution-NoDerivs 3.0 Unported|	PetalArt|
 |server.png|[Link](https://findicons.com/icon/185522/network_server)|GNU Lesser General Public License|silvestre herrera|
 |service.png|[Link](https://findicons.com/icon/227231/server)|GNU Lesser General Public License|David Vignoni|
-|network.png|[Link](https://findicons.com/icon/169709/globe)|GNU Lesser General Public License|YellowIcon|
-|person.png|[Link](http://www.iconarchive.com/show/ecommerce-business-icons-by-designcontest/admin-icon.html)|CC Attribution 4.0|DesignContest|
+|star.png|[Link](http://www.softicons.com/web-icons/web-2-icon-set-by-anders-bjarnle/star-icon)|Free Use of icon|Anders Bjarnle|
+|tick.png|[Link](http://www.softicons.com/system-icons/crystal-clear-icons-by-everaldo-coelho/actions-agt-action-success-icon)|GNU Lesser General Public License |Everaldo Coelho|
+|trash.png|[Link](http://www.softicons.com/business-icons/ecommerce-and-business-icons-by-designcontest.com/trash-icon)|CC Attribution 3.0 Unported|DesignContest.com|
```

Comment:
It looks like this commit also adds `trash.png` to the `README.md`? If so, could the commit message be amended to mention that?

Reply:
Will do.